packaging/debian-sid/patches/0016-skip-TestPopulateFromSeedWithConnectHook.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/overlord/devicestate/firstboot_test.go b/overlord/devicestate/firstboot_test.go
+index 47ac7a7a90..19b505673b 100644
+--- a/overlord/devicestate/firstboot_test.go
++++ b/overlord/devicestate/firstboot_test.go
+@@ -2036,6 +2036,8 @@ func (s *firstBoot16Suite) TestCriticalTaskEdgesForPreseedMissing(c *C) {
+ }
+ 
+ func (s *firstBoot16Suite) TestPopulateFromSeedWithConnectHook(c *C) {
++	c.Skip("not converging on debian")
```

Comment:
The reason this test is failing on Debian is missing mocking and the Debian network sandbox. I've fixed it here https://github.com/snapcore/snapd/pull/12385 - you can drop this patch once this lands.

Reply:
Thanks, added a comment